src/components/book.js

```diff
@@ -1,24 +1,24 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Kindly remove this linter disabling here as previously required. You fix the resulting linter errors by doing the following:

  - Import PropTypes here
     ```
     import PropTypes from 'prop-types';
     ```
  - Just as in the suggestion above add this validations tworads the end of the file:
    ```
    Book.propTypes = {
      title: PropTypes.string.isRequired,
      author: PropTypes.string.isRequired,
      id: PropTypes.string.isRequired,
    };
    ```

Reply:
Thank you so much for your guidance.